website/content/docs/k8s/connect/observability/metrics.mdx

```diff
@@ -112,7 +112,7 @@ Consul’s built-in UI has a topology visualization for services part of the Con
 
 The diagram below illustrates how the UI displays service metrics for a sample application:
 
-[![UI Topology View](/img/ui-topology.png)](/img/ui-topology.png)
+[![UI Topology View](/img/ui-service-topology-view-hover.png)](/img/ui-service-topology-view-hover.png)
```

Comment:
Is there a reason for why this being wrapped inside a `[]` as a whole, then followed by the parentheses (path) ?

Reply:
good question, I just left it how i found it and didn't realize, it has now been updated.